content/post/2017-08-04-redefine-statistical-significance.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 统计的显著性水平
+author: 战立侃
+date: '2017-08-04'
+slug: redefine-statistical-significance
+categories:
+tags:
+---
+ 
+
+
+提高某些领域研究可重复性的一个直接和简单的方法是提高假设检验的统计显著性水平。
+
+# 1. 统计检验和显著性水平
+
+统计检验中，研究者最终希望确定零假设 (`$H_{i,i=0}$`) 和备则假设 (`$H_{i, i \neq 0}$`)  中哪一个正确。由于这两种状态的不相容性，事实情况只有两种：要么是零假设正确；要么是其中一个备择假设如 `$H_1$` 正确。让事情变得更为复杂的是，研究者用于判断哪个假设正确的证据并不来自于数据整体，而来自于整体数据的一部分，即一个样本。样本统计量对于总体参数来说始终存在一定偏差，即抽样误差。
```

Comment:
“要么是其中一个备择假设”，一共有几个备择假设？
数据整体，整体数据。建议统一词汇。
为什么样本统计量对于总体参数始终存在一定偏差？？

Reply:
下面是维基百科关于取样和取样偏差的描述，我说的是“样本统计量对于总体参数始终存在一定程度的偏差”，问题在什么地方呢？
In statistics, sampling error is the error caused by observing a sample instead of the whole population.[1] The sampling error is the difference between a sample statistic used to estimate a population parameter and the actual but unknown value of the parameter.[2]

Reply:
偏差，英文是bias。误差，英文是error。根本不是一个东西啊。
样本统计量只能说存在抽样误差。偏差，是没有的。

Reply:
我举个直观简单的例子翻译 @SpinozaRUC 的话

```r
set.seed(2018)
x <- rnorm(100,0,1)
mean(x)
```

样本均值是样本统计量中的一个，上述三行R代码实现一次抽样过程，显然，样本均值 0.02119194 不是 0，即使增大样本量肯定还是不等于 0， 这就是抽样带来的误差。只有将样本量取到无穷这个极限，样本均值才会等于总体均值，即 0 。

但是样本均值对于总体均值而言是无偏的，因为这是一个无偏估计
<img src="http://latex.codecogs.com/png.latex?E(\bar{x}-\mu)&space;=&space;0&space;\\&space;where,\bar{x}&space;=&space;\sum_{i=1}^{n}x_{i}&space;\\&space;x_{1},x_{2},\cdots,x_{i}&space;\overset{i.i.d}{\sim}&space;N(\mu,\sigma^2)" title="E(\bar{x}-\mu) = 0 \\ where,\bar{x} = \sum_{i=1}^{n}x_{i} \\ x_{1},x_{2},\cdots,x_{i} \overset{i.i.d}{\sim} N(\mu,\sigma^2)" />